src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/V62/Eth62ProtocolHandler.cs

```diff
@@ -266,7 +266,7 @@ protected void Handle(TransactionsMessage msg)
 
         private ValueTask HandleSlow((IOwnedReadOnlyList<Transaction> txs, int startIndex) request, CancellationToken cancellationToken)
         {
-            IOwnedReadOnlyList<Transaction> transactions = request.txs;
+            using IOwnedReadOnlyList<Transaction> transactions = request.txs;
```

Comment:
See line 279. It could be that the handling is cancelled in the middle because of block processing for example. So you can't dispose this list as it will get used later with a different startIndex.

Reply:
fixed